src/Gui/NotificationBox.cpp

```diff
@@ -190,20 +206,17 @@ void NotificationLabel::resizeEvent(QResizeEvent *e)
     QLabel::resizeEvent(e);
 }
 
-NotificationLabel::~NotificationLabel()
-{
-    instance = nullptr;
-}
 void NotificationLabel::hideNotification()
 {
     if (!hideTimer.isActive()) {
         hideTimer.start(300);
     }
 }
+
 void NotificationLabel::hideNotificationImmediately()
 {
     close(); // to trigger QEvent::Close which stops the animation
-    deleteLater();
+    instance = nullptr;
```

Comment:
Not sure I'm correct on this, but how about `instance.reset()` ?

Reply:
It is the same:
https://stackoverflow.com/questions/16151550/c11-when-clearing-shared-ptr-should-i-use-reset-or-set-to-nullptr

Reply:
Great. So my preference would be to use 'reset'. Personal taste only. :)

Reply:
I prefer the nullptr assignment myself. I consider it more expressive. Just a matter of taste. :)